### ISSUE TITLE
When I use the optimizer with the distributed tensorflow synchronization gradient, I used the keras.optimizer.SGD optimizer inside keras, but did the optimizer support tensorflow synchronous distribution?

### ISSUE THREAD

**Issue by guoying1030** [fchollet/keras#5774]
When I use the optimizer with the distributed tensorflow synchronization gradient, I used the keras.optimizer.SGD optimizer inside keras, but did the optimizer support tensorflow synchronous distribution?

what should i do??
